Controls/Input/Input.UWP/Calendar/View/RadCalendar.cs

```diff
@@ -2060,11 +2371,36 @@ private static void OnDisplayModePropertyChanged(DependencyObject target, Depend
 
             RadCalendar calendar = (RadCalendar)target;
 
-            calendar.model.DisplayMode = (CalendarDisplayMode)args.NewValue;
+            calendar.displayModeCache = (CalendarDisplayMode)args.NewValue;
+            calendar.model.DisplayMode = calendar.displayModeCache;
 
             calendar.UpdateNavigationHeaderContent();
             calendar.UpdateNavigationPreviousNextButtonsState();
 
+            if (calendar.navigationPanel != null)
+            {
+                if (calendar.displayModeCache == CalendarDisplayMode.MultiDayView)
+                {
+                    if (calendar.navigationPanel.previousButton != null && calendar.navigationPanel.nextButton != null)
+                    {
+                        calendar.navigationPanel.previousButton.Content = "\xE012";
```

Comment:
the same button symbols (**"\xE012"** for example) are used in CalendarNavigationControl - extract a constant so we can easily change them